packages/react-components/react-tabster/src/hooks/useMergeTabsterAttributes.ts

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Types } from 'tabster';
+
+/**
+ * Merges a collection of tabster attributes.
+ *
+ * ⚠️The attributes passed as arguments to this  hook cannot change at runtime.
+ * @internal
+ * @param attributes - collection of tabster attributes from other react-tabster hooks
+ * @returns single merged tabster attribute
+ */
+export const useMergedTabsterAttributes_unstable: (
+  ...attributes: Types.TabsterDOMAttribute[]
+) => Types.TabsterDOMAttribute = (...attributes) => {
```

Comment:
This looks so overcomplicated...

Reply:
This is exactly what is done in Teams, there is no better way to merge tabster attributes, it's a significant amount of tech debt we have in react-tabster and we've indicated that it should be changed. But it's not very high on our backlog right now